public/css/common.less

```diff
@@ -212,15 +212,37 @@ div.show-more {
   }
 }
 
+.footer {
+  div:first-child:nth-last-child(2) ~ div {
```

Comment:
I don't quite get what this is meant for. Could you explain this or better, simplify it?

Reply:
Since you told me to use custom containers for these selected elements, I had to do a lot of research on how to style the footer without changing the previous style. I then found out that you can style it any way you want based on the number of elements.  This line only changes the style of the footer if two div elements are added, otherwise it keeps the previous style. Why I do this is because now we add and remove a new container with JS which we need for these selected elements as well as a fixed container for the status bar, then this technique is very handy. I couldn't find any simplified other solution unfortunately.

Reply:
If you would consider that as simplified, it also works that way:
```css
div:nth-last-child(2) ~ div ...
```

Reply:
I'd rather make the entire footer a flex container instead of the `.status-bar`.

So, add `display: flex;` to the footer. `flex: 1 1 auto` to the `.selection-count`. `margin-left: auto;` to the `.status-bar`.

You should also remove all flexbox related styles from `.status-bar` and apply `display: inline-block;` to the contained `.state-bagdes` list instead.

This way your selection count is also on the very left and the previous layout keeps unchanged. It's also more readable and on point.

Reply:
Indeed, the two dives then don't get the same width but this way also works. Ty.